elections.md

```diff
@@ -101,4 +87,15 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Bootstrap recusal
+
+Special note: The bootstrap committee pledges to recuse itself from any direct
```

Comment:
Moved this down from the top because it seems less important now

Reply:
I would just rename this to "steering committee" as the bootstrap committee is gone and now just part of the SC.

Reply:
Will do